src/management-system-v2/db-helper.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# Configurable parameters
+CONTAINER_NAME="postgres_database_proceed"  # Replace with your Docker container name
+POSTGRES_USER="proceed"                     # PostgreSQL user
+POSTGRES_PASSWORD="proceed"                 # PostgreSQL password
+DEFAULT_DB="proceed_db"                     # The default database name when --default is used
+ENV_FILE="./.env"                           
+DB_HOST="localhost"
+DB_PORT="5432"
+DEFAULT_SCHEMA="public"
+
+# Parse command-line arguments
+USE_DEFAULT_DB=false
+CREATE_NEW_DB=false
+DELETE_DB=false
+
+for arg in "$@"; do
+  case $arg in
+    --default)
+      USE_DEFAULT_DB=true
+      ;;
+    --new)
+      CREATE_NEW_DB=true
+      ;;
+    --delete)
+      DELETE_DB=true
+      ;;
+    *)
+      echo "Unknown option: $arg"
+      exit 1
+      ;;
+  esac
+done
+
+# Get the current branch name
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+# Replace hyphens with underscores in the branch name
+BRANCH_NAME_SAFE=$(echo "$BRANCH_NAME" | tr '-' '_')
```

Comment:
Is slash in the branch name a problem?

Reply:
you cannot create a db with "-" character in the name

Reply:
But the slash would be a problem in the env URL, right?

Reply:
stimmt, wir müssen slash vermeiden in db name